src/Compilers/CSharp/Portable/Binder/Semantics/OverloadResolution/OverloadResolution.cs

```diff
@@ -2105,19 +2105,19 @@ private static ParameterSymbol GetParameter(int argIndex, MemberAnalysisResult r
                 return (m1ModifierCount < m2ModifierCount) ? BetterResult.Left : BetterResult.Right;
             }
 
-            // Otherwise, prefer methods with 'val' parameters over 'in' parameters.
-            return PreferValOverInParameters(arguments, m1, m1LeastOverriddenParameters, m2, m2LeastOverriddenParameters);
+            // Otherwise, prefer methods with 'val' parameters over 'in' parameters and over 'ref' parameters when the argument is an interpolated string handler.
```

Comment:
This needs to be documented in the spec, but I'd propose we do so.

Reply:
//PROTOTYPE to track?

---
In reply to: [638121526](https://github.com/dotnet/roslyn/pull/53602#discussion_r638121526) [](ancestors = 638121526)